activities.go

```diff
@@ -0,0 +1,34 @@
+package subscription
+
+import (
+	"context"
+	"fmt"
+
+	"go.temporal.io/sdk/activity"
+)
+
+type Activities struct{}
+
+func (a *Activities) SendWelcomeEmail(ctx context.Context, subscription SubscriptionInfo) error {
```

Comment:
```suggestion
func (*Activities) SendWelcomeEmail(ctx context.Context, subscription SubscriptionInfo) error {
```

Unused receiver variable (and really don't even need pointer receiver, but meh)